x/distribution/keeper/keeper.go

```diff
@@ -181,12 +189,12 @@ func (k Keeper) WithdrawValidatorCommission(ctx context.Context, valAddr sdk.Val
 		return nil, err
 	}
 	// update outstanding
-	outstanding, err := k.GetValidatorOutstandingRewards(ctx, valAddr)
+	outstanding, err := k.ValidatorOutstandingRewards.Get(ctx, valAddr)
```

Comment:
here we have a behavioural change for which in case there are no outstanding rewards we error.
this is to make it impossible to subtract zero coins.